Checkout/Checkout/Source/Validation/Constants.swift

```diff
@@ -25,7 +25,7 @@ enum Constants {
   }
 
   enum Product {
-    static let version = "0.1.0"
```

Comment:
Do we know why this was 0.1.0 ?

Reply:
I don't know why it was 0.1.0 .

Parent enum was renamed from `Network` as part of the migration for Checkout package to Frames.

Value is being used in:
- request headers to network: reported as User Agent so probably for logging purposes by backend
- checkout logging

I am unsure whether this was 0.1.0 before, as there seems to be a large commit made by Harry in May, moving the Checkout module inside Frames 🤷‍♂️. Overall imo it would be better to sync it with rest of versioning infos (unless we have good reason not to?). Sandbox is unaffected in completing tokenisation requests with this change.